src/client/helpers/formatter.js

```diff
@@ -1,11 +1,11 @@
-export const jsonParse = str => {
+export function jsonParse(input) {
+  if (!input) return null;
```

Comment:
isn't that covered by the catch?

Reply:
This also makes function return `null` for input of `false` which is not desired.

Reply:
ok, didn't expect JSON.parse to behave like this... `JSON.parse(0) => 0` and `JSON.parse(false) => false` shouldn't this throw an exception? anyway...